<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
    background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(34, 148, 157, 255), stop:0.890052 rgba(255, 177, 177, 255));
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="welcome_label">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;Ubuntu&quot;;</string>
        </property>
        <property name="text">
         <string>Welcome</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="img_button">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	font: 15pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border: 1px solid transparent;
    color: #FFFFFF; /* Couleur du texte */
}

QToolButton:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond lorsqu'il est survolé */
}

QToolButton:pressed {
    background-color: rgba(0, 0, 0, 0.4); /* Couleur de fond lorsqu'il est cliqué */
}</string>
          </property>
          <property name="text">
           <string>Image file</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="img_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
    width: 23px;
    height: 23px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QToolButton" name="seg_button">
          <property name="styleSheet">
           <string notr="true">QToolButton {
	font: 15pt &quot;Ubuntu&quot;;
    background-color: transparent;
    border: 1px solid transparent;
    color: #FFFFFF; /* Couleur du texte */
}

QToolButton:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond lorsqu'il est survolé */
}

QToolButton:pressed {
    background-color: rgba(0, 0, 0, 0.4); /* Couleur de fond lorsqu'il est cliqué */
}</string>
          </property>
          <property name="text">
           <string>Segmentation file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="seg_box">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox::indicator {
    width: 23px;
    height: 23px;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="continue_button">
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: transparent;
    border: 1px solid transparent;
    color: #FFFFFF; /* Couleur du texte */
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.2); /* Couleur de fond lorsqu'il est survolé */
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.4); /* Couleur de fond lorsqu'il est cliqué */
}

</string>
        </property>
        <property name="text">
         <string>Continue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
